sql/policy_internal.sql

```diff
@@ -48,7 +48,7 @@ AS $$
 DECLARE
   htoid       REGCLASS;
   chunk_rec   RECORD;
-  numchunks   INTEGER := 1;
+  numchunks_compressed   INTEGER := 1;
```

Comment:
I think the initial value should be zero, no?

Reply:
fixed.